library/coretests/tests/cell.rs

```diff
@@ -477,3 +478,42 @@ fn const_cells() {
         const _: i32 = CELL.into_inner();
     */
 }
+
+#[allow(dead_code)]
+fn const_refcell() {
```

Comment:
Please also call `borrow` and `borrow_mut` to model the usual usage (and, in particular, to ensure their destructors work).

Reply:
Done